### DISCUSSION TITLE
UI inconsistence between mobile and web/desktop

### DISCUSSION THREAD

**Discussion by simondubrulle** [ente-io/ente#4034] [General]
Hi, I am really enjoying Ente Photos and I noticed some inconsistences when it comes to UI. I was wondering if this was a deliberate choice (technical limitation) or if this can be "improved". I feel like it would improve user experience if all the UIs share as much as possible.

I will simply list what I have found so far:

## WHEN SELECTING A PHOTO IN TIMELINE/ALBUM

| | ON MOBILE (ANDROID, v0.9.58) | ON WEB/DESKTOP |
| --- | --- | --- |
| Add to album | ✅ | ✅ |
| Favorite | ✅ | ✅ |
| Guest view | ✅ | ❌ |
| Delete | ✅ | ✅ |
| Hide | ✅ | ✅ |
| Archive | ✅ | ✅ |
| Download  | ✅ | ✅ |
| Share | ✅ | ❌ (but platform specific) |
| Send link (creates a shareable link) | ✅ | ❌ |
| Fix time | ❌ | ✅ |
| Edit location | ✅ | ❌ |

## WHEN LOOKING AT A PHOTO (FULL VIEW)
| | ON MOBILE (ANDROID, v0.9.58) | ON WEB/DESKTOP |
| --- | --- | --- |
| Display info | ✅ | ✅ |
| Edit | ✅ | ✅ |
| Delete | ✅ | ✅ |
| Favorite | ✅ | ✅ |
| Download  | ✅ | ✅ |
| Hide | ✅ | ❌ |
| Archive | ✅ | ❌ |
| Play motion video (if any) | ✅ | ❌ |
| Set as | ✅ | ❌ (but platform specific) |
| Guest view | ✅| ❌ (but platform specific) |
| Share | ✅ | ❌ (but platform specific) |
| Toggle fullscreen | ❌ (but platform specific) | ✅ |
| Zoom in/out | ❌ (but platform specific) | ✅ |
| Copy as PNG | ❌ (but platform specific) | ✅ |
| Send link (creates a shareable link) | ❌ | ❌ |
| Fix time | ❌ | ❌ |
| Edit location | ❌ | ❌ |
| Add to album | ❌ | ❌ |

**Comment by simondubrulle** [comment]
Is there any reason why, for example, we cannot edit location on Web/Desktop, or cannot fix time on Mobile ? Or why can't we add photo to an album from the full view of a photo?

**Comment by devguilhermy** [comment]
Really nice overview @simondubrulle!
I am missing this consistency as well
